packages/drivers/odsp-driver-definitions/src/tokenFetch.ts

```diff
@@ -12,6 +12,13 @@ export interface TokenResponse {
 	/** Token value */
 	token: string;
 
+	/**
+	 * Authorization header value will be used verbatim when making network call that requires the token.
+	 * If not returned the token value will be assumed to be a Bearer token and will be used to generate the
```

Comment:
Nit: should this say "if not provided"? Not sure I understand "returned" in this context.